examples/example02_trip98.py

```diff
@@ -50,16 +50,23 @@
 # we may print all contours found in the Vdx file, if we want to
 print(v.get_voi_names())
 
-# Ok, we have the Contours and the CT cube ready. Next we must prepare a plan.
+# Prepare Kernel data for a given projectile:
+mykernel = pte.KernelModel()
+mykernel.projectile = pte.Projectile("C", a=12)
+mykernel.ddd_path = "/home/bassler/TRiP98/base/DATA/DDD/12C/RF3MM/*"
+mykernel.spc_path = "/home/bassler/TRiP98/base/DATA/SPC/12C/RF3MM/*"
+mykernel.sis_path = "/home/bassler/TRiP98/base/DATA/SIS/12C.sis"
+mykernel.rifi_thickness = 3.0  # 3 mm ripple filter. (Only for documentaiton, will not affect dose optimization.)
+mykernel.rifi_name = "GSI_1D_3mm"  # Additional free text for documentation.
+mykernel.comment = "Carbon-12 ions with 3 mm 1D Ripple Filter"
+
+# Ok, we have the Contours, the CT cube and dose kernels ready. Next we must prepare a plan.
 # We may choose any basename for the patient. All output files will be named using
 # this basename.
-plan = pte.Plan(basename=patient_name)
+plan = pte.Plan(basename=patient_name, kernels=[mykernel])
```

Comment:
Using a tuple  `kernels=(mykernel,)` would be more elegant (and less error prone) - we don't wish to modify it in function call.

Reply:
IMHO it looks less readable, due to that trailing comma.